pkg/operator/mon/util.go

```diff
@@ -302,3 +302,13 @@ func extractKey(contents string) (string, error) {
 	}
 	return secret, nil
 }
+
+func getNodeId(node *NodeInfo) (string, error) {
```

Comment:
this function doesn't look like it can return an error at all, so the return type should probably just be string